sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/MathExpressionsSuite.scala

```diff
@@ -403,11 +403,13 @@ class MathExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
 
   test("sqrt") {
     testUnary(Sqrt, math.sqrt, (0 to 20).map(_ * 0.1))
-    testUnary(Sqrt, math.sqrt, (-5 to -1).map(_ * 1.0), expectNaN = true)
+    testUnary(Sqrt, math.sqrt, (-5 to -1).map(_ * 1.0), expectNull = true)
```

Comment:
Yea, I was writing this comment. If `NaN` makes sense in a way, I was thinking we couldn't consider this case as a bug that should be fixed.